transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/auth/BasicLoginModuleTestRoleIT.java

```diff
@@ -1,83 +0,0 @@
-/**
- * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.kaazing.gateway.transport.wsn.auth;
-
-import static org.kaazing.test.util.ITUtil.createRuleChain;
-
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TestRule;
-import org.kaazing.gateway.server.test.GatewayRule;
-import org.kaazing.gateway.server.test.config.GatewayConfiguration;
-import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
-import org.kaazing.k3po.junit.annotation.Specification;
-import org.kaazing.k3po.junit.rules.K3poRule;
-
-public class BasicLoginModuleTestRoleIT {
```

Comment:
Hi,

This was deleted by @dpwspoon in [this commit](https://github.com/kaazing/gateway/pull/631/commits/e00d24bf9cc6b123ec0b6776d473297e6978d786) with the comment: 
Revert "Login Module fix"

Regards,
Doru